mobify/mobify-code-style
# AMP code style

PR: https://mobify.atlassian.net/browse/AMP-32

## Changes
- Remove ignore styles in base folder.

## How To Test
- Go to: https://github.com/mobify/platform-scaffold/pull/658
- Follow "How to test-drive this PR" 
- Make sure there is no lint error 

## TODOs:
~Updated README~
